nengo/networks/circularconvolution.py

```diff
@@ -195,8 +193,9 @@ def CircularConvolution(n_neurons, dimensions, invert_a=False, invert_b=False,
     with net:
         net.A = nengo.Node(size_in=dimensions, label="A")
         net.B = nengo.Node(size_in=dimensions, label="B")
-        net.product = Product(n_neurons, tr_out.shape[1],
-                              input_magnitude=input_magnitude * 2)
+        net.product = Product(
+            n_neurons, tr_out.shape[1],
+            input_magnitude=input_magnitude * 2. / np.sqrt(dimensions))
```

Comment:
This changes an assumption underlying this network. Without the `sqrt(dimensions)` scaling each input dimension is regarded as uniformly distributed. With the scaling the input vectors are assumed to be unit length (times `input_magnitude`). It seems to be unlikely (or at least much rarer) that this network is used in cases where the first assumption holds true. I'm not entirely sure what the purpose of the 2 factor is.